irohad/model/converters/pb_command_factory.hpp

```diff
@@ -131,6 +130,11 @@ namespace iroha {
             const model::Command &command);
         std::shared_ptr<model::Command> deserializeAbstractCommand(
             const protocol::Command &command);
+
+       private:
+        boost::bimap<iroha::protocol::RolePermission, std::string> pb_role_map_;
```

Comment:
Why do we need complex keys (::bimap) in this place?

Reply:
It allows to get rid of duplication. Only writing one map instead of two

Reply:
ok